### DISCUSSION TITLE
[FR] Display total number of sales and revenue (Net)

### DISCUSSION THREAD

**Discussion by ben-callaway** [craftcms/console#91] [Craft Console]
Currently, there isn't a way to view how many sales of a plugin in Craftnet, you can only see active installs. As a tweak to https://id.craftcms.com/developer/sales you could show top level information (see screenshot). Clicking number of sales (or possibly plugin title or other CTA) would take you to the current page, but filtered by the plugin selected.

![sales](https://user-images.githubusercontent.com/7083462/44487379-0f59bf80-a64e-11e8-9b37-09340c0f9e8b.png)

**Comment by benjamindavid** [comment]
A lot of great ideas there, thank you! The sales section just got improved, where sales can now be filtered by customer email or plugin name. We will be adding more filtering options along the way.
